### ISSUE TITLE
% breaks compilation

### ISSUE THREAD

**Issue by Try2Code** [gpoore/minted#323]
hi!

I encountered a compilation error that seems to get triggered by simple tex comment starting with '%'

```tex
\documentclass{beamer}
\usepackage{minted}
\begin{document}
\begin{frame}[fragile]
\frametitle{Python Code listing in Beamer}
\begin{minted}{python}
    print('The sum of {0} and {1} is {2}'.format(num1, num2, sum))
\end{minted}
\end{frame}%                                                                                                                                           
\end{document}
```
Putting '%' at the end of an existing line (l.6,8,9) leads to an error. putting it to a separate line does not lead to an error. Is this a known limitation? is this a parsing issue? I thought comments like this are normal latex syntax - am I right?

**Comment by Try2Code** [comment]
the point is that frames need to be `fragile` in order to use minted. So the conclusion is to avoid comments like this in frames with source code, right?

**Comment by gpoore** [comment]
Yes, you can't have comments like this with fragile frames, at least with the way that `beamer` currently works.

**Comment by Try2Code** [comment]
Alright - thx a lot for the fast support @gpoore @muzimuzhi
